tests/group_vars/all/all

```diff
@@ -1,5 +1,4 @@
 ---
-streisand_ci: yes
```

Comment:
Why is this being removed?

Reply:
@cpu Because it's already passed in `vars_ci.yml`. Vars passed by `-e / --extra-vars` have the highest priority: http://docs.ansible.com/ansible/latest/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable

Reply:
If you added the vars from `vars_ci.yml` here could you leave this as-is? Does that work or is there something about the priority adjustment that your PR requires?

Reply:
@cpu This could be left here as-is. It's just a habit to clean up duplicate definitions.